frontend/src/components/Navbar.jsx

```
@@ -38,7 +38,7 @@ const Navbar = () => {
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
     >
-      <button className={styles.menuItem} onClick={() => useNav("/entry")}>
+      <button className={styles.menuItem} onClick={() => navigate("/entry")}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)